src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -253,6 +253,8 @@ install_packages_in_root (RpmOstreeTreeComposeContext  *self,
   {
     GPtrArray *sources;
     JsonArray *enable_repos = NULL;
+    gs_unref_hashtable GHashTable *repos_to_enable =
+      g_hash_table_new (g_str_hash, g_str_equal);
```

Comment:
The variable transitions from `repos_to_enable` at the start to `missing_repo_names` at the end...so it's kind of both.  But I think I can clean this up...